Refactor docs (#156)
rstudio/rstudio-docker-products: helper/float/README.md

> # Floating license server

Reviewer: I don't think we should be making this image public (relatively weak opinion, to be clear). 

Seems obvious it's useful for our testing, but I don't think we want customers setting up license servers for testing they're likely to foul up, especially since they'll need new license keys...

Author: For sure - I'm inclined to leave public (they have been for a long while and we've never had questions) because it is possible that we could build these containers in such a way that they could end up being useful (we do not build / push them anywhere) and a valid way to run floating licensing. 

I will definitely remove the link from the README though!

Reviewer: Can we add a stronger (and very clear) disclaimer? Something like, "This image is for testing only. You are probably in the wrong place unless an RStudio employee told you to use this repository."

**WARNING**: Floating Licenses should not be used within docker containers in a production context, since the docker
container failing could require manual intervention to fix (see the app below). We provide these images only for

> development and testing.

Author: @akgold Does this warning not suffice? Not clear enough?

Reviewer: I think it's ok. I'd probably vote to go even a little stronger. I struggle to see a situation where customers should use these at all if they're not in direct conversation with us.

Like...I think our preferred recommendation would be to set up the license server, then start playing with the rest of Team.

Author: For sure. I agree 😄 I'll make it stronger

If you want to test floating licenses locally. You will need to do the following:

> - Request a floating license key (these are different from normal license keys)

Reviewer: ```suggestion
- Request a floating license key from support@rstudio.com
```

> - Ensure that the floating license key works within a hypervisor (by default, they do not)

Reviewer: How would I ensure this?

Author: You'd ask your support / CS rep 😄

Reviewer: Can we just say that then?
